apigw-rest-api-dynamodb-lambda-cdk/README.md

```diff
@@ -0,0 +1,144 @@
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd apigw-rest-api-dynamodb-lambda-cdk/cdk
+    ```
+
+1. Install dependencies
+    ```
+    npm install
+	cd src
+	pip install aws-xray-sdk --target .
+	pip install aws_lambda_powertools --target .
+    ```
+
+1. Deploy the stack to your default AWS account and region. The output of this command should give you the HTTP API URL.
+    ```
+    cdk deploy
+    ```
+
+## How it works
+
+This pattern creates an Amazon API Gateway REST API that integrates with an Amazon DynamoDB table. The API integrates directly with the DynamoDB API and supports [PutItem](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_PutItem.html) and [Query](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html) actions. The API Integration has read and write access to the DynamoDB. There is no authentication on the API endpoint
+
+## Testing
+
+Upon deployment, you will see the API endpoint URL in the output. It will take the format:
+
+`https://${API_ID}.execute-api.${REGION_NAME}.amazonaws.com`
+
+Once the application is deployed, use [Postman](https://www.postman.com/) to test the API using the following instructions.
+
+1. Launch Postman
+
+1. Invoke the DynamoDB **PutItem** action to add a new item to the DynamoDB table:
+	* Enter the API URL with the **prod** stage as the path:.
+	```
+	https://${API_ID}.execute-api.${REGION_NAME}.amazonaws.com/prod/awsomedynamodb
+	```
+	* Select **POST** as the HTTP method from the drop-down list to the left of the address bar.
+	* Choose the **Body** tab. Choose **raw** and select **JSON** from the drop-down list. Enter the following into the text box: 
+	```
+	{
+		"id":"14",
+    	"name":"Do some thing",
+    	"description":"Do some thing",
+    	"customer":"John"
+	}
+	```
+	* Choose **Send** to submit the request and receive a "200 OK" response.
+	* Open the DynamoDB console and select the table which was created to confirm that the item has been added.
+	* Change the values for `pk` or `data` in the POST body and repeat this process to add multiple items to the DynamoDB table.
+
+1. Invoke the DynamoDB **Query** action to query all items by artist in the DynamoDB table:
+	* Enter the Invoke URL in the address bar. Add **/prod/foo** to the URL path.
+	* Add **/foo** to the URL path. This defines the ID that you want to query.
+	```
+	https://${API_ID}.execute-api.${REGION_NAME}.amazonaws.com/prod/awsomedynamodb
+	```
+	* Select **GET** as the HTTP method from the drop-down list to the left of the address bar.
+	* Choose the **Body** tab. Choose **none**.
+	* Choose **Send** to submit the request and receive a "200 OK" response with a list of the matching results. Example: 
+	```
+		{
+			"Count": 1,
+			"Items": [
+				{
+					"pk": {
+						"S": "foo"
+					},
+					"data": {
```

Comment:
The `pk` and `data` are not items you are putting in the previous section.  Please check your intputs

Reply:
Updated